src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/MockHttpClientTransport.cs

```diff
@@ -370,4 +369,10 @@ public enum CheckKind
             }
         }
     }
+
+    static class Utf8
```

Comment:
I'd just remove the helper especially if it is just used a couple times, it doesn't seem to be worth the indirection.

Reply:
I removed one of them. The other I think we need as we add more tests, we will use it in more place.